CONTRIBUTING

```diff
@@ -0,0 +1,84 @@
+# Contribution Guidelines
+
+The following documents constitutes a set of guidelines to which contributors must adhere.
+
+* [Contributions Licensing](#contributions-licensing)
+* [Developer Certificate of Origin](#developer-certificate-of-origin)
+* [Issues and Support](#issues-and-support)
+
+## Contributions Licensing
+
+Any contribution that you make to this repository will
+be under the Apache 2 License, as dictated by that
+[license](http://www.apache.org/licenses/LICENSE-2.0.html):
+
+~~~
+5. Submission of Contributions. Unless You explicitly state otherwise,
+   any Contribution intentionally submitted for inclusion in the Work
+   by You to the Licensor shall be under the terms and conditions of
+   this License, without any additional terms or conditions.
+   Notwithstanding the above, nothing herein shall supersede or modify
+   the terms of any separate license agreement you may have executed
+   with Licensor regarding such Contributions.
+~~~
+
+## Developer Certificate of Origin
+
+Contributors must sign-off each commit by adding a `Signed-off-by: ...`
+line to commit messages to certify that they have the right to submit
+the code they are contributing to the project according to the
+[Developer Certificate of Origin (DCO)](https://developercertificate.org/).
+
+## Issues and Support
+
+*eProsima Fast DDS* developers welcome all contributions, and we will be grateful if you follow the guidelines below while contributing to this project.
+Firs of all, there are several products related with Fast DDS.
+Please, open the issue in the corresponding repository:
+
+- [Fast DDS](https://github.com/eProsima/Fast-DDS)
+- [Fast DDS documentation](https://github.com/eProsima/Fast-DDS-docs/issues/new)
+- [Fast DDS-Gen](https://github.com/eProsima/Fast-DDS-Gen/issues/new)
+- [Fast DDS Python](https://github.com/eProsima/Fast-DDS-python)
+- [Foonathan memory vendor](https://github.com/eProsima/foonathan_memory_vendor/issues/new)
+- [Fast CDR](https://github.com/eProsima/Fast-CDR/issues/new)
+- [Shapes Demo](https://github.com/eProsima/ShapesDemo/issues/new)
+- [ROS 2 Fast DDS RMW](https://github.com/ros2/rmw_fastrtps/issues/new)
+- [Fast DDS Statistics Backend](https://github.com/eProsima/Fast-DDS-statistics-backend/issues/new)
+- [Fast DDS Monitor](https://github.com/eProsima/Fast-DDS-monitor/issues/new)
+- [DDS Router](https://github.com/eProsima/DDS-Router/issues/new)
+
```

Comment:
Should we include here the [Fast DDS QoS Profiles Manager](https://github.com/eProsima/Fast-DDS-QoS-Profiles-Manager) ?

Reply:
I do not think so. That project is best left out and forgotten :see_no_evil: